src/lib/elementary/meson.build

```
@@ -949,6 +963,10 @@ elementary_src = files([
   'efl_ui_spotlight_animation_manager.c',
 ])
 
+if get_option('ector')
+  elementary_src += 'efl_ui_vg_animation.c'
+endif
```

Does it work? Shouldn't it be:
```suggestion
if get_option('ector')
  elementary_src += files(['efl_ui_vg_animation.c'])
endif
```